scripts/restart_ecs

```diff
@@ -3,6 +3,24 @@
 # Restart ECS service
 #
 
+healthcheck_url="https://${APP_ENV}.easi.cms.gov/api/v1/healthcheck"
+healthcheck_timeout="180"
+
 cluster_name="${APP_ENV}-easi-app"
 service_name="easi-app"
 ( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment )
+
+# wait for the new service to come up
+echo -n "Waiting for health check version to match new deployment." 1>&2
```

Comment:
More of an open question: With the time between the call to `aws ecs update-service` and the first call to the health check url, can we be confident that the new task is stable? I suppose the new task is already successfully registered with the load balancer and healthy or else the curl command to the health check url would still be retrieving the commit hash from the old task.

Reply:
I think we can be confident that it is not stable when we run the first check. The idea is to give the health check a bit of time to begin reporting the expected version. If it is still reporting an unexpected version at the end of that window, we fail the build.

Reply:
Yeah, I misread the output in CI at first. I thought it had successfully retrieved the new commit hash on the first attempt, which seemed odd. But now I see the script did go through a few loop iterations.